install/kubernetes/cilium/templates/hubble/dashboards-configmap.yaml

```
@@ -0,0 +1,29 @@
+{{- if .Values.hubble.metrics.dashboards.enabled }}
+{{- $files := .Files.Glob "files/hubble/dashboards/*.json" }}
+{{- if $files }}
+apiVersion: v1
+kind: ConfigMapList
+items:
+{{- range $path, $fileContents := $files }}
+{{- $dashboardName := regexReplaceAll "(^.*/)(.*)\\.json$" $path "${2}" }}
```

nit: strange to see the start of string anchor inside the first capture group, also why two capture groups if we only extract one?
```suggestion
{{- $dashboardName := regexReplaceAll "^.*/(.*)\\.json$" $path "${1}" }}
```

---

To be honest, I copied this from https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml and I didn't really look too deeply at this.